test-data/unit/check-incremental.test

```diff
@@ -1025,10 +1025,7 @@ import a.b
 
 [file a/b.py]
 
-[rechecked b]
-[stale]
-[out2]
-tmp/b.py:4: error: Name "a" already defined on line 3
+[stale b]
```

Comment:
Not fully sure what's going on with this particular test case, maybe we should delete it since the behaviour looks pretty similar to ignore. Alternatively I could add some new special case logic in checker.py to disallow assignments via `import x` to things typed Any.